fw/http_frame.c

```diff
@@ -1308,7 +1307,8 @@ do {									\
 		if (hdr->flags & HTTP2_F_PADDED)
 			return tfw_h2_recv_padded(ctx);
 
-		SET_TO_READ_VERIFY(ctx, HTTP2_RECV_DATA);
+		SET_TO_READ(ctx);
```

Comment:
We need to ignore frames for LOC_CLOSED state

Reply:
Good point.